cypress/commons/actions/generic/Login.js

```diff
@@ -37,9 +37,12 @@ function login(options) {
 
       let reqAuthAlias;
       let browseCallback;
-      if (auth.USE_SERVICE_ACCOUNT) {
+      if (auth.USE_SERVICE_ACCOUNT || auth.USE_API_KEY) {
```

Comment:
Same question as usual, is there any chance to have error with attribute as _false_ instead of _null/undefined_ ?

Reply:
Both constants `USE_SERVICE_ACCOUNT` and `USE_API_KEY` are defined in the _authUtils_ module, they are boolean values returned by `Cypress.env('<some env var>') === 1;`. Unless this code changes, they can't be `null` nor `undefined`